cli/src/commands/mod.rs

```diff
@@ -581,6 +584,10 @@ struct MoveArgs {
 /// If the source became empty and both the source and destination had a
 /// non-empty description, you will be asked for the combined description. If
 /// either was empty, then the other one will be used.
+///
+/// Note that if squashing the working-copy commit results in it getting abandoned,
+/// a new empty commit will still be created as the working-copy commit.
```

Comment:
We be good to use "a new empty working-copy commit will be created" to match the text above